### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader(NVDA):NVDA does not annouce the the selected imgae as selected.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/AzureStorageExplorer#2023]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#MAS1.3.1,#DesktopApp;#electron.0,#NVDA
**Environment Details:** 
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2
**MAS References:**MAS1.3.1
**Steps to Reproduce :**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster” .
5.	Start NVDA and observe the announcement.
**Actual:**
When the focus is going on the selected image or the radio button the NVDA does not announce the selected word for the selected image.
****Expected:**
NVDA should announce the selected item as selected.

**User Impact:**
Screen reader User will not able to understand on which server they are connected to.

 
MAS Reference:
MAS1.3.1 - 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_AsureDataStudio_Selectthedeploymentoption_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3706587/A11y_AsureDataStudio_Selectthedeploymentoption_ScreenReader.pptx)

**Comment by jinglouMSFT** [comment]
Wrong place to log this bug.
